multiples.py

```diff
@@ -15,6 +16,14 @@ def good(i):
     return by_3 or by_5
 
 
+def even_elements_mult(i):
+    result = 1
+    for num in i:
```

Comment:
Are you sure you can use for loops like that? Is it a valid way to define a for loop?

Reply:
yes

Reply:
I googled it